src/app/services/bucket/dummyBucketService.py

```diff
@@ -0,0 +1,16 @@
+from app.services.bucket.baseBucketService import BaseBucketService
+
+
+class DummyBucketService(BaseBucketService):
+    def __init__(self):
+        """Dummy class used for testing purpose.
+        """
```

Comment:
Same here, no need to specify the constructor

And also, for class docstring, it's better to put it before (outside) the constructor. Automatic documentation system won't catch it if it's placed in the constructor